### PR TITLE
SAP Monitor/SapHana: Add error checks

### PR DESCRIPTION
**Pull request by tniek**
This PR checks certain error conditions (mostly related to the upgrade to v2.0) and prints out meaningful messages to the users.
Error conditions checked include:
- Custom log table `SapHana_LoadHistory_CL` does not exist yet
image
- Custom log table `SapHana_LoadHistory_CL` exists, but it doesn't have a column `HOST_s` that was already used in v1.5 **--or--**
Custom table `SapHana_LoadHistory_CL` exists with columns `HOST_s` and `PROVIDER_INSTANCE_s`, but doesn't have any records yet
image
- Custom log table `SapHana_LoadHistory_CL` exists with column `HOST_s`, but it is missing a new column `PROVIDER_INSTANCE_s` introduced in v2.0
image

## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [x] *if updating templates, it is helpful to post a screenshot of what the changes look like.*
* [x] *if updating templates, ensure that your parameters and steps have meaningful names.*
* [ ] *if adding new templates, or changing items in a gallery, it's helpful to post a screenshot of what the gallery looks like now*

### CODE REVIEW

File: Workbooks/SapMonitor/SapHana/SapHana.workbook [hunk 1]
```diff
@@ -21,19 +21,115 @@
...
...= \"HOST_s\";\nlet c2 = \"PROVIDER_INSTANCE_s\";\nlet dummy = \"unmöglicher-wert!\";\nlet MissingTable = view () { print tableMissing = 1, columnV1 = dummy, columnV2 = dummy, records = -1 };\nlet checkRecords =\n  union\n    isfuzzy=true MissingTable,\n    (table(t)\n      | summarize records = count()\n    )\n  | project records\n  | top 1 by records desc\n;\nlet checkExists =\n  union\n    isfuzzy=true MissingTable,\n    (table(t)\n      | extend tableMissing = 0\n      | extend columnV1 = column_ifexists(c1, dummy)\n      | extend columnV2 = column_ifexists(c2, dummy)\n    )\n  | project tableMissing, columnV1, columnV2\n  | limit 2\n  | top 1 by tableMissing asc\n;\nlet tableExists = toscalar(checkExists | summarize max(tableMissing) != 1);\nlet columnV1Exists = toscalar(checkExists | summarize max(columnV1) != dummy);\nlet columnV2Exists = toscalar(checkExists | summarize max(columnV2) != dummy);\nlet tableRecords = toscalar(checkRecords | summarize max(records));\nlet x = iif(tableExists, iif(columnV1Exists, iif(columnV2Exists, iif(tableRecords > 0, \"0\", \"2\"), \"3\"), \"2\"), \"1\");\nprint toscalar(x)\n//print tableExists, columnV1Exists, columnV2Exists, tableRecords\n",
```
Comment (persiaAziz) [inline Workbooks/SapMonitor/SapHana/SapHana.workbook:30]: dummy = \"unmöglicher-wert!\" ??

Comment (persiaAziz) [inline Workbooks/SapMonitor/SapHana/SapHana.workbook:30]: How does this column V1 v2 render on errors? DO they get replaced by the actual column name?

Comment (tniek) [reply Workbooks/SapMonitor/SapHana/SapHana.workbook:30]: Some hardcore Kusto magic going on here...
Basically setting dummy to an "impossible value" (that's what that means).

Comment (tniek) [reply Workbooks/SapMonitor/SapHana/SapHana.workbook:30]: It's built in such a way that they always have a value, regardless whether the column (or even the underlying table) exists.

**Pull request merged**